modules/gds_mongodb/manifests/init.pp

```diff
@@ -1,12 +1,20 @@
 # class to configure a replica set
-class gds_mongodb($members, $replSet) {
+class gds_mongodb($members, $replSet, $enable_mongo3_repo = false) {
+
   file { '/etc/mongodb':
     ensure => 'directory',
     owner  => 'root',
     group  => 'root',
     mode   => '0644',
   }
 
+  if $enable_mongo3_repo {
+    include ::gds_mongodb::repo::mongodb3
+    include ::mongodb::client
```

Comment:
I can't figure out why we need to include this new class now, but only for Mongo 3.